skillserver/skillserver.go

```diff
@@ -291,14 +291,26 @@ func IsValidAlexaRequest(w http.ResponseWriter, r *http.Request) bool {
 }
 
 func readCert(certURL string) ([]byte, error) {
-	cert, err := http.Get(certURL)
+	certPool, err := x509.SystemCertPool()
+	insecureSkipVerify := false
+	if err != nil || certPool == nil {
+		log.Println("Can't open system cert pools, doing insecure skip verify instead")
```

Comment:
this seems dangerous to me. I think a better option would be to either take a new argument which allows `readCert` to skip ssl verification OR keep this logic but, if ssl verification is skipped but a cert is retrieved, return a typed error to indicate this. The second option maintains backwards compatibility the best. Consumers of `readCert` function could then check the error type and proceed (or not) if they care about ssl verification being skipped.

Reply:
Please see above, thanks